lib/helpers/errors/generic.js

```diff
@@ -0,0 +1,26 @@
+'use strict'
+
+class GenericError extends Error {
+  constructor (message, { code = 'ERR_GENERIC', details } = {}) {
+    super(message)
+
+    this.status = 500
+    this.code = code
+
+    if (details) {
+      this.details = details
+    }
+
+    Error.captureStackTrace(this)
+  }
+
+  toJsonResponse () {
```

Comment:
I think there's a helper method named toJSON that is called automatically if exists during the call JSON.stringify(myobject). Not sure if it would help in res.json case but would be helpful for serialization in other possible mediums. So maybe just renaming toJsonResponse into toJSON would help here, but ofc it's good also as it is now :)

Reply:
hey, I haven't found [toJSON](https://nodejs.org/api/errors.html) standard method

I named the method `toJsonResponse` to emphasize that it contains only response-specific data

Reply:
hey @batrudinych, it's not specific to Error type but in general it's used by JSON.stringify method internally https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/JSON/stringify#toJSON_behavior